pmdtester.gemspec

```diff
@@ -0,0 +1,63 @@
+  s.homepage = "https://github.com/pmd/pmd-regression-tester".freeze
+  s.licenses = ["BSD-2-Clause".freeze]
+  s.rdoc_options = ["--main".freeze, "README.rdoc".freeze]
+  s.rubygems_version = "2.6.14".freeze
+  s.summary = "A regression testing tool ensure that new problems and unexpected behaviors will not be introduced to PMD project after fixing an issue , and new rules can work as expected.".freeze
+
+  if s.respond_to? :specification_version then
+    s.specification_version = 4
+
+    if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
+      s.add_runtime_dependency(%q<nokogiri>.freeze, ["~> 1.8.2"])
+      s.add_runtime_dependency(%q<slop>.freeze, ["~> 4.6.2"])
+      s.add_development_dependency(%q<hoe-bundler>.freeze, ["~> 1.2"])
+      s.add_development_dependency(%q<hoe-git>.freeze, ["~> 1.6"])
+      s.add_development_dependency(%q<minitest>.freeze, ["~> 5.10.1"])
+      s.add_development_dependency(%q<mocha>.freeze, ["~> 1.5.0"])
+      s.add_development_dependency(%q<rubocop>.freeze, ["~> 0.56.0"])
+      s.add_development_dependency(%q<test-unit>.freeze, ["~> 3.2.3"])
+      s.add_development_dependency(%q<rdoc>.freeze, ["< 6", ">= 4.0"])
+      s.add_development_dependency(%q<hoe>.freeze, ["~> 3.17"])
+    else
+      s.add_dependency(%q<nokogiri>.freeze, ["~> 1.8.2"])
+      s.add_dependency(%q<slop>.freeze, ["~> 4.6.2"])
+      s.add_dependency(%q<hoe-bundler>.freeze, ["~> 1.2"])
+      s.add_dependency(%q<hoe-git>.freeze, ["~> 1.6"])
+      s.add_dependency(%q<minitest>.freeze, ["~> 5.10.1"])
+      s.add_dependency(%q<mocha>.freeze, ["~> 1.5.0"])
+      s.add_dependency(%q<rubocop>.freeze, ["~> 0.56.0"])
+      s.add_dependency(%q<test-unit>.freeze, ["~> 3.2.3"])
+      s.add_dependency(%q<rdoc>.freeze, ["< 6", ">= 4.0"])
+      s.add_dependency(%q<hoe>.freeze, ["~> 3.17"])
+    end
+  else
+    s.add_dependency(%q<nokogiri>.freeze, ["~> 1.8.2"])
+    s.add_dependency(%q<slop>.freeze, ["~> 4.6.2"])
+    s.add_dependency(%q<hoe-bundler>.freeze, ["~> 1.2"])
+    s.add_dependency(%q<hoe-git>.freeze, ["~> 1.6"])
+    s.add_dependency(%q<minitest>.freeze, ["~> 5.10.1"])
+    s.add_dependency(%q<mocha>.freeze, ["~> 1.5.0"])
+    s.add_dependency(%q<rubocop>.freeze, ["~> 0.56.0"])
+    s.add_dependency(%q<test-unit>.freeze, ["~> 3.2.3"])
+    s.add_dependency(%q<rdoc>.freeze, ["< 6", ">= 4.0"])
+    s.add_dependency(%q<hoe>.freeze, ["~> 3.17"])
+  end
+end
+
+# DO NOT EDIT THIS FILE. Instead, edit Rakefile, and run `rake hoe:spec`.
```

Comment:
I'd add this comment at the beginning of the file, rather at the end.
I'll change this, when I merge it.